src/api/posts/package.json

```
@@ -4,7 +4,10 @@
   "private": true,
   "description": "A service for retrieving posts",
   "scripts": {
-    "start": "node src/server.js"
+    "start": "node src/server.js",
+    "eslint": "TIMING=1 eslint --config .eslintrc.js \"**/*.{js,jsx}\"",
```

We don't have jsx files in this service. Also, the file pattern should be `./**/*` so that it doesn't include files outside of the current folder.
```suggestion
    "eslint": "TIMING=1 eslint --config .eslintrc.js \"./**/*.js\"",
```